bigquery_etl/format_sql/format.py

```diff
@@ -141,6 +141,7 @@
     "sql/moz-fx-data-shared-prod/udf_js/jackknife_mean_ci.sql",
     "sql/moz-fx-data-shared-prod/udf_js/jackknife_ratio_ci.sql",
     "sql/moz-fx-data-shared-prod/udf_js/jackknife_sum_ci.sql",
+    "sql/moz-fx-data-shared-prod/udf_js/jackknife_percentile_ci/udf.sql",
```

Comment:
Is the `udf.sql` at the end of the path necessary? The row above doesn't have this included in the path, though the SQL is defined in a `udf.sql` files.